java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -46,6 +51,15 @@ class StorageService : ResurrectorService() {
     override fun onCreate() {
         super.onCreate()
         startTime = startTime ?: System.currentTimeMillis()
+
+        val periodicCleanupTask =
+                PeriodicWorkRequest.Builder(PeriodicCleanupTask::class.java, 1, TimeUnit.HOURS)
+                        .addTag(PeriodicCleanupTask.WORKER_TAG)
+                        .build()
```

Comment:
Continuation indentation in kotlin is only 4 spaces (one tab), not two-tabs like in java.

Reply:
done